nc/source/ecs/EcsApiImpl.cpp

```diff
@@ -19,11 +19,21 @@ namespace nc
         return CreateEntity(Vector3::Zero(), Quaternion::Identity(), Vector3::One(), internal::DefaultEntityTag);
     }
 
+    EntityHandle CreateEntity(Vector3 pos)
+    {
+        return CreateEntity(pos, Quaternion::Identity(), Vector3::One(), internal::DefaultEntityTag);
```

Comment:
Doesn't strictly pertain to this review, and Github does not let me leave review comments on unmodified code, but it would be a fun side project to create a little helper function that generates a friendly default name per class type. (ie, internal::DefaultEntityTag -> Point Light)